app/webpacker/src/javascripts/help_popup.js

```diff
@@ -0,0 +1,49 @@
+$("#close_box").on("click", function(event) {
+  hide_help_popup();
+})
+
+$("#close_box_longer").on("click", function(event) {
+  setCookie("show_digital_assistant_popup", "disable", 28);
+  hide_help_popup();
+})
+
+function show_help_popup() {
+  $("#help_popup").fadeIn(1000);
+}
+
+function hide_help_popup() {
+  $("#help_popup").css("display", "none");
+}
+
+setTimeout(checkCookieAndBrowser, 30000);
+
+function checkCookieAndBrowser() {
+  let cookieValue = getCookie("show_digital_assistant_popup");
+  var isMobile = /Android|webOS|iPhone|iPad|iPod|BlackBerry|IEMobile|Opera Mini/i.test(navigator.userAgent)
+
+  if (cookieValue == "" && isMobile == false) {
+    show_help_popup();
+  }
+}
+
+function setCookie(cname, cvalue, exdays) {
+  const d = new Date();
+  d.setTime(d.getTime() + (exdays * 24 * 60 * 60 * 1000));
+  let expires = "expires="+d.toUTCString();
+  document.cookie = cname + "=" + cvalue + ";" + expires;
+}
+
+function getCookie(cname) {
```

Comment:
are sure we need to write our own function to get a cookie,
I'd expect some functionality present for it (maybe some js lib ?)